src/snowflake/snowpark/dataframe_reader.py

```diff
@@ -1205,13 +1213,13 @@ def _to_snowpark_type(self, schema: Tuple[tuple]) -> StructType:
             fields.append(field)
         return StructType(fields)
 
-    def upload_and_copy_into_table(
+    def _upload_and_copy_into_table_with_rename(
```

Comment:
I'm just curious why there is a `_rename` in the signature?

Reply:
probably mistyped, fixing